app/Jobs/GeneratePackageOpenGraphImage.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Jobs;
+
+use GDText\Box;
+use GDText\Color;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Storage;
+
+class GeneratePackageOpenGraphImage implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    protected $packageName;
+    protected $packageAuthor;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(String $packageName, String $packageAuthor)
```

Comment:
How hard would it be to pass an instance of the Package model instead of the individual strings here? The one nice thing about that is that then we could add the open graph image name code (with the Str::slug) as a method on the package.

Reply:
_or_, its third param could be the OG name, and then this could become even dumber